src/sass/components/_header.scss

```diff
@@ -222,3 +222,108 @@ header {
     }
   }
 }
+
+// media queries for mobile
+@media screen and (min-width: 320px) and (max-width: 480px) {
+  header {
+    .top-bar {
+      ul {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+
+        li {
+          display: inline-block;
+          line-height: 55px;
+
+          a {
+            color: #ffffff;
+            font-size: 13px;
+            margin-right: 5px;
+
+            &:hover {
+              color: $primary;
+              text-decoration: none;
+            }
+          }
+        }
+      }
+
+      .text-left {
+        padding-left: 5px;
+        padding-right: 0;
+        ul {
+          display: flex;
+          justify-content: space-around;
+          li {
+            margin-right: 0;
+            i {
+              margin-left: 2px;
+            }
+            .top-bar-menu-el {
```

Comment:
to i pozostałe zagnieżdżanie skutkuje selektorami typu `header .top-bar .text-left ul li .top-bar-menu-el`, które są bardzo skomplikowane i trudne do nadpisania.

staraj się trzymać zagnieżdżenia na poziomie 2-3 poziomów (oczywiście nie jest to żelazna zasada, ale jest to dobra "baza"), w razie potrzeby definiuj konkretne klasy dla elementów. upraszcza to wynikowy kod i ułatwia nadpisywanie.

Reply:
Okej, dodałem pomocniczą klasę.
**Pytanie:** Jeśli są zmiany w każdym elemencie w drzewku to niestety nie zrobię 2-3 poziomów. Chyba, że mam zaczynać od nowa, albo pomijać element główny tzn.
- `header .top-bar`
- `header .top-bar-menu .tob-bar-span` etc (przykład)

Nie wiem, czy wiesz o co mi chodzi. Nie ma jednego drzewka tylko jest więcej mniejszych i pomijanie elementów, których właściwości się nie zmieniają.

Reply:
to zalezy od specificity.

jesli selektory są napisane np. jako `.first-class .second-class tag tag tag` to mozesz to nadpisać np. przy pomocy `.first-class .second-class .third-class`, bo specificity tego drugiego selektora bedzie wyzsze.

dodatkowo jesli widzisz problemy (np. wielokrotnie zagnieżdżone elementy) to nic nie stoi na przeszkodzie by odrobinę zrefaktorować style w tym zakresie i uprościć ich zagnieżdżanie.

ale to jest rada na przyszłość.

linki 
https://specificity.keegan.st/
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity